modules/core/src/main/scala/org/tessellation/l0/SnapshotStorage.scala

```diff
@@ -0,0 +1,25 @@
+package org.tessellation.l0
+
+import cats.effect.kernel.{Async, Ref}
+import cats.syntax.functor._
+
+import org.tessellation.schema.height.Height
+
+trait SnapshotStorage[F[_]] {
+  def storeSnapshot(snapshot: DAGSnapshot): F[Unit]
+  def getLastSnapshotHeight: F[Height]
+}
+
+object SnapshotStorage {
+
+  def make[F[_]: Async]: SnapshotStorage[F] =
+    new SnapshotStorage[F] {
+      private val lastSnapshot: Ref[F, Option[DAGSnapshot]] = Ref.unsafe(None)
```

Comment:
Please don't use `Ref.unsafe`

Reply:
Sure. I will make it `F[SnapshotStorage[F]]`